src/Extensions/Hosting.Services.Web/Middlewares/ObsoleteCorrelationHeadersMiddleware.cs

```diff
@@ -20,7 +20,7 @@ namespace Microsoft.Omex.Extensions.Hosting.Services.Web.Middlewares
 	/// This middleware should be removed after all services moved to use <see cref="Activity"/>
 	/// </remarks>
 	[Obsolete("Use it only if you need to comunicate with services that use old correlation", false)]
-	internal class ObsoleteCorrelationHeadersMiddleware
+	public class ObsoleteCorrelationHeadersMiddleware
```

Comment:
Doesn't InvokeAsync now need null guards added, since it will now be consumed externally?

Reply:
You are right :( Only acceptable usage for these classes if for application builder:
`builder.UseMiddleware<ObsoleteCorrelationHeadersMiddleware>()`, but by making them public there is no way of enforcing it, I'll create extension methods for adding middlewares and will make all classes internal.

Reply:
Done, now all of them are internal and there are extension methods to register them

Reply:
@AndreyTretyak I'm not seeing any changes to the access modifiers.

Reply:
@K-Cully sorry it looks like I have not pushed changes, should be there now